pages/form/[formId]/create.tsx

```diff
@@ -0,0 +1,15 @@
+import WorkspacesLayout from '@/layouts/Workspaces';
+import { NextPage } from 'next';
+import { useRouter } from 'next/router';
+
+const CreateForm: NextPage = () => {
+  const { query } = useRouter();
+
+  return (
+    <WorkspacesLayout>
```

Comment:
This will need to be a new EditFormLayout or something similar, we'll get to that when we get to editing forms